#non-git version (without .git directory) build fails
#bundled kiva version is tied to enable
#Somewhere in the code it is buried to read version info from .git directory. 
#I just made kiva use the version of enable
#The inspiration was taken from 
#commit_ids 51e11cc f91c824 https://github.com/enthought/enable
# https://github.com/enthought/enable/commit/f91c824ebad74e8c23e62fe3390842c5fe322aee
#https://github.com/enthought/enable/commit/f91c824ebad74e8c23e62fe3390842c5fe322aee
#Up to now, this commits are only merged to the maintenance 4.5.x branch.
#if you clone the the git repositiry, you do not need this patch.
#kiva is icluded as well as agg for kiva in enable.i
#As far as I know, Enthought does not distribute a standalone kiva,i
# at the homepage of this package referred to have two parts,
# enable and kiva (see http://code.enthought.com/projects/enable/)
#Aug 24th 2016 by B. Hajgato (Free University Brussels - VUB)
--- enable-168e222dababd4464062eb7a3ab92c4b591f7031/setup.py.org	2016-06-14 15:25:39.000000000 +0200
+++ enable-168e222dababd4464062eb7a3ab92c4b591f7031/setup.py	2016-08-24 13:50:19.959208391 +0200
@@ -93,18 +87,19 @@
     version = full_version
 """
     # Adding the git rev number needs to be done inside
-    # write_version_py(), otherwise the import of kiva._version messes
+    # write_version_py(), otherwise the import of enable._version messes
     # up the build under Python 3.
     fullversion = VERSION
     if os.path.exists('.git'):
         git_revision, dev_num = git_version()
-    elif os.path.exists('kiva/_version.py'):
+    # All packages are synced with the enable version
+    elif os.path.exists('enable/_version.py'):
         # must be a source distribution, use existing version file
         try:
-            from kiva._version import git_revision, full_version
+            from enable._version import git_revision, full_version
         except ImportError:
             raise ImportError("Unable to import git_revision. Try removing "
-                              "kiva/_version.py and the build directory "
+                              "enable/_version.py and the build directory "
                               "before building.")
 
         match = re.match(r'.*?\.dev(?P<dev_num>\d+)', full_version)
